plugins/http_plugin/include/eosio/http_plugin/beast_http_session.hpp

```diff
@@ -140,9 +140,6 @@ class beast_http_session : public detail::abstract_conn {
             return;
          }
 
-         // verfiy bytes in flight/requests in flight
-         if(!verify_max_bytes_in_flight()) return;
```

Comment:
I think I prefer this to be left as is without the addition of a passed in size. The idea here is to stop new requests after the threshold has been exceeded. Once you have done the work to create the large string you might as well send it.

Reply:
The check is done in both branches of the following `if` statement, either before calling `send_response()` or in `make_http_response_handler()`

Reply:
We had a nice discussion and Greg convinced me the his approach here is better.